src/licensedcode/data/licenses/latex-ec-fonts.LICENSE

```diff
@@ -0,0 +1,81 @@
+% copyrite.txt
```

Comment:
This is a new license alright! @DennisClark what id would you see best for this one?

Reply:
ecfonts-2012

Reply:
wait, better: 

ecfonts-1.0